Remove target reference constraint on Coverage.subscriber

alexzautke opened the pull request:
Fixes #466 and https://service.gematik.de/browse/PTDATA-1950

alexzautke committed:
Remove target reference constraint on Coverage.subscriber

alexzautke committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali reviewed (commented):
todo: releasenotes und als TC aufsetzen

alexzautke committed:
Update ReleaseNotes for ISiK Bericht Bundle changes

Added new slices for Patient and Encounter in ISiK report bundle and fixed harmonization with German base profile 'CoverageDeGkv'.
